CantoboardFramework/Keyboard/View/KeyCap.swift

```diff
@@ -366,60 +366,38 @@ indirect enum KeyCap: Equatable, ExpressibleByStringLiteral {
         case ">": return [">", "〉", "＞"]
         case "«": return ["«", "《"]
         case "»": return ["»", "》"]
-        case "•": return ["•", "·", "．", "°"]
-        // #+= 3rd row
+        case "&": return ["＆", "&", "§"]
+        case "·": return ["·", "．", "•", "°"]
+        // #+= 4th row
         case "…": return ["…", "⋯"]
-        // 123 1st row full width
```

Comment:
Why are we removing them?

Reply:
Because there aren’t any fullwidth number keys on the keyboard.

Reply:
Here I’m removing unused and abundant children key caps.

Reply:
Because there are no fullwidth number keys on the keyboard; these lines are not in use.